spec/features/tasks_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'feature_helper'
+
+describe "Tasks" do
+
+  before(:each) do
+    # FIXME - Having to set this really high for the 'adds a task to a story
+    # spec'.  Need to work on making it more responsive.
+    Capybara.default_max_wait_time = 10
+    sign_in user
+  end
+
+  let(:user)        { create :user, :with_team, email: 'user@example.com', password: 'password' }
+  let(:project)     { create(:project, name: 'Test Project', users: [user], teams: [user.teams.first] ) }
```

Comment:
Or remove the `(` from here, or add it on the previous `let`. Just to keep the same pattern. 
The name is already on the factory https://github.com/Codeminer42/cm42-central/blob/master/spec/factories.rb#L32, don't need it here.